certified-connectors/Actsoft/apiDefinition.swagger.json

```diff
@@ -2,29 +2,26 @@
   "swagger": "2.0",
   "info": {
     "version": "1.0",
-    "title": "Actsoft Mobile Workforce Plus",
-    "description": "Actsoft Mobile Workforce Plus is an all-in-one service management platform that helps you more effectively manage your remote teams of employees and general business operations away from the office. As a cloud-based solution, WFM enables remote data collection and displays it via its web app dashboard, providing managers with the additional insight they need to create smarter strategies for increased efficiency, productivity, and reduced costs while coordinating their mobile workforces.",
+    "title": "TeamWherxs",
```

Comment:
@rdameron Please let us the know reason for updating your connector? It is not recommended by our infrastructure team as it may break existing flows created for your connector,

Reply:
Ron put as reply to whole conversation instead of this thread

From Ron:
Our marketing department rebranded the application from Mobile Workforce
Plus to be named TeamWherx.

From James to elaborate:
The name was changed by our marketing department right after we released the first version of this, I do not believe there is any usage of this connector yet it is brand new